src/CoreLib/deploy.fs

```diff
@@ -139,7 +139,7 @@ type DeploymentSettings() =
             drives |> Seq.find( fun drive -> not (DeploymentSettings.ExcludedDrives.ContainsKey( drive )) )
     static member val internal GetFirstDataDrive = DeploymentSettings.GetFirstDataDriveImpl with get, set
     /// Directory at remote node that holds Prajna operating state. 
-    static member val LocalFolder = Path.Combine(DeploymentSettings.RootFolder, "OneNet") with get
+    static member val LocalFolder = Path.Combine(DeploymentSettings.RootFolder, "Prajna") with get
```

Comment:
For test clusters with existing data, on each machine, I will create a symbolic link Prajna that points to original folder